Lib/pathlib/_abc.py

```diff
@@ -806,21 +822,12 @@ def _copy_metadata(self, target, *, follow_symlinks=True):
             metadata = self._read_metadata(keys, follow_symlinks=follow_symlinks)
             target._write_metadata(metadata, follow_symlinks=follow_symlinks)
 
-    def copy(self, target, *, follow_symlinks=True, preserve_metadata=False):
+    def _copy_data(self, target):
```

Comment:
I worry ``_copy_data`` is too broad/vague, especially given this takes a `Path` rather than `bytes` or `memoryview` or &c. The obvious choice is `_copy_file` but perhaps there's a reason for avoiding that?